source/lambda/apiprocessor/redaction.py

```diff
@@ -131,10 +130,23 @@ def validateHeadersFooters(items, typeStr):
                     
                     
 def validateRedactedItem(item):
-                    
-    if len(item.keys()) != REDACTED_ITEM_KEYS:
+    
+    numOfkeys = len(item.keys())
+    
+    # the default number of mandatory keys
+    if numOfkeys == REDACTED_ITEM_NUM_KEYS:
+        pass
+    # case of the optional label key added, validate it here
+    elif numOfkeys == (REDACTED_ITEM_NUM_KEYS + 1):
```

Comment:
Can we create this "1" as a constant as well (for maybe OPTIONAL_ITEM_NUM_KEYS)?

Reply:
yes, actually let's make it more straightforward and define 2 constants instead, as this validation code.